lh-test/src/main/java/io/littlehorse/test/LHExtension.java

```diff
@@ -46,16 +47,19 @@ public void postProcessTestInstance(Object testInstance, ExtensionContext contex
                 }
                 store.put(worker.getTaskDefName(), worker);
                 worker.registerTaskDef(true);
+                TaskDefId taskDefId =
+                        TaskDefId.newBuilder().setName(worker.getTaskDefName()).build();
                 Awaitility.await()
-                        .ignoreException(LHMisconfigurationException.class)
-                        .until(() -> {
-                            try {
-                                worker.start();
-                                return true;
-                            } catch (IOException e) {
-                                throw new IllegalStateException(e);
-                            }
-                        });
+                        .ignoreExceptionsMatching(exn -> LHTestExceptionUtil.isNotFoundException(exn))
```

Comment:
```suggestion
                        .ignoreExceptionsMatching(LHTestExceptionUtil::isNotFoundException)
```

Reply:
For some reason, that didn't compile. I got a red underline on my editor.